autonomy/fsm/scaffold/generators/components.py

```diff
@@ -24,7 +24,7 @@
 
 from aea.protocols.generator.common import _camel_case_to_snake_case
 
-from autonomy.fsm.scaffold.base import AbstractFileGenerator
+from autonomy.fsm.scaffold.base import AbstractFileGenerator, _indent_wrapper
```

Comment:
No need to import this if you use the `template_kwargs` in the base instead.

Reply:
done